sklearn/tests/test_metaestimators.py

```diff
@@ -55,7 +55,9 @@ def __init__(
         skip_methods=["score"],
     ),
     DelegatorData("RFE", RFE, skip_methods=["transform", "inverse_transform"]),
-    DelegatorData("RFECV", RFECV, skip_methods=["transform", "inverse_transform"]),
+    DelegatorData(
```

Comment:
I know this isn't part of your PR necessarily, but I'm unsure what `DELEGATING_METAESTIMATORS` purpose is, and can't easily figure it out from the glancing at the code.

Is it worth adding some comments somewhere above to indicate what's going on here?

cc: @adrinjalali

Reply:
Yeah this is cryptic unless one reads the actual test. We're checking if these methods are only available if the sub-estimator has them, and since now we're implementing the `score` w/o caring about the sub-estimator, we can skip it here. But a comment somewhere wouldn't hurt.

Reply:
Added a comment.